bin/firecrown

```diff
@@ -14,21 +14,26 @@ def run(action, config):
     Also, run some ACTION on a CONFIG file."""
     config, data = firecrown.parse(config)
     print("Watch out! Here comes a firecrown!")
-    print("config file:\n", pprint.pformat(config))
 
     # FIXME: stubby code for testing
     if action == 'compute':
         params = {}
         for p in ['Omega_k', 'Omega_b', 'Omega_c', 'h',
-                  'n_s', 'sigma8', 'w0', 'wa']:
+                  'n_s', 'A_s', 'w0', 'wa']:
             if p in config['parameters']:
-                params[p] = config['parameters'][p]
+                val = config['parameters'][p]
+                if isinstance(val, list):
+                    val=val[1]
+                params[p] = val
 
         cosmo = pyccl.Cosmology(**params)
         loglike, stats = firecrown.compute_loglike(cosmo=cosmo, data=data)
         print('loglike:', loglike)
         print('stats:', pprint.pformat(stats))
 
+    elif action == 'sample':
```

Comment:
Let's call this 'run-cosmosis'. It is more descriptive.

Reply:
That's only useful if there's a second sampling approach in there.  It's really not a good idea to have a whole second implementation.

Reply:
I agree that two sampling approaches may be problematic. I’m also not happy having infrastructure split across three or more repos but this may be a fact of life. So from my point of view it is not clear yet what is going to happen. 

Let’s change the name to be more descriptive and then shelve this discussion for the time being.

Reply:
Name changed, and I've updated the readme to reflect this.